x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/tour.tsx

```diff
@@ -19,7 +19,7 @@ import { securityTourConfig, SecurityStepId } from './tour_config';
 export interface TourContextValue {
   activeStep: number;
   endTourStep: (stepId: SecurityStepId) => void;
-  incrementStep: (stepId: SecurityStepId, step?: number) => void;
+  incrementStep: (stepId: SecurityStepId) => void;
```

Comment:
never used/implemented

Reply:
Is it going to be used in the future or should we just remove it?

Reply:
oh ignore me